### PR TITLE
feat: add code to fetch client_credentials grant token

### PR DESCRIPTION
**Pull request by brownoxford**
Ref: w3c-ccg/traceability-interop#34

### CODE REVIEW

**Review by mprorock — APPROVED** [review]

File: tests/postman/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "commander": "^7.2.0",
     "is-url-superb": "^6.1.0",
     "ky-universal": "^0.9.1",
+    "md5": "^2.3.0",
```
Comment (OR13) [inline tests/postman/package.json:25]: sadge

Comment (mprorock) [reply tests/postman/package.json:25]: sha would be better

**Review by mprorock — CHANGES_REQUESTED** [review]
Per discussion on call - will add auth into one newman call

**Comment by brownoxford** [comment]
Will also add note to documentation that retrieval of auth token may trigger billable event.

File: tests/postman/package.json (hunk 1 above)
Comment (brownoxford) [reply tests/postman/package.json]: Updated to replace MD5 with SHA256 from crypto-js module

File: tests/postman/README.md [hunk 2]
```diff
@@ -130,6 +130,8 @@ To import the collections into Postman, follow the guide on [importing Postman d
 
 ## OAuth 2.0 Configuration
 
+Postman tests make use of OAuth 2.0 client credentials flow for authentication, which may result in a billable event for configured authentication providers.
```
Comment (TallTed) [inline tests/postman/README.md:133]:
```suggestion
_**NOTE:** Postman tests make use of OAuth 2.0 client credentials flow for authentication, which **may result in a billable event** for configured authentication providers._
```

**Review by OR13 — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Pull request merged**